### DISCUSSION TITLE
Easily switch the base model

### DISCUSSION THREAD

**Discussion by forever-lwy** [open-webui/open-webui#6048] [General]
# Feature Request

**Important: Before submitting a feature request, please check whether a similar issue or feature request has already been posted in the Issues or Discussions section. It's likely we're already tracking it. In case of uncertainty, initiate a discussion post first. This helps us all to efficiently focus on improving the project.**

**Let's collaborate respectfully. If you bring negativity, please understand our capacity to engage may be limited. If you're open to learning and communicating constructively, we're more than happy to assist you. Remember, Open WebUI is a volunteer-driven project maintained by a single maintainer, supported by our amazing contributors who also manage full-time jobs. We respect your time; please respect ours. If you have an issue, We highly encourage you to submit a pull request or to fork the project. We actively work to prevent contributor burnout to preserve the quality and continuity of Open WebUI.**

**Is your feature request related to a problem? Please describe.**
Sometimes I need to switch between different base models while retaining system prompts and other settings to try out different results. In such cases, it becomes cumbersome to enter the workspace and make modifications. If I use the system prompts for conversations, I have to re-enter them every time I start a new conversation, and I can't switch between multiple prompts.

**Describe the solution you'd like**
In the chat interface, users should be able to directly switch between base models or save certain system prompts. These saved prompts can then be easily selected and switched within the chat interface.

**Describe alternatives you've considered**

**Additional context**

**Comment by tjbck** [comment]
You can edit or clone the model.

**Reply by forever-lwy** [reply]
Thank you for your reply. I have tried the method you mentioned. However , directly modifying the base model is quite troublesome as it requires entering the workspace. As for copying a model and selecting a different base model, if you need to modify the system prompts later, you'd have to change them for all models, which is even more cumbersome.
